kolibri/utils/options.py

```diff
@@ -130,6 +129,12 @@ def calculate_thread_pool():
             "default": False,
             "envvars": ("KOLIBRI_SERVER_PROFILE",),
         },
+        "DEBUG": {"type": "boolean", "default": False, "envvars": ("KOLIBRI_DEBUG",)},
+        "DEBUG_DATABASE": {
+            "type": "boolean",
+            "default": False,
+            "envvars": ("KOLIBRI_DEBUG_DATABASE",),
+        },
```

Comment:
Why does `DEBUG_DATABASE` need to be a different option than `DEBUG`? I'm uncomfortable with this.

* It sets a for adding new `DEBUG_*` flags
* We need to be careful to keep the number of options small and essential, because everything in here comes with maintenance and support costs
* The name `DEBUG_DATABASE` is a bit confusing, and makes me thing I'm setting a value for database used for debugging.

Instead, we could keep things simpler and do:

```python
LOGGING = get_logging_config(conf.LOG_ROOT, debug=DEBUG, debug_database=DEBUG)
```

Reply:
Good idea with a different name for it, so it doesn't get too confusing.

But outputting all database queries is extremely noisy, far more noisy than existing debugging. I had to separate it out to make use of the normal `--debug` option.

Reply:
Suggesting `[KOLIBRI_]DEBUG_LOG_DATABASE` ?

Reply:
> It sets a precedent for adding new

Agreed, but it's unfortunately the only way I can see this becoming convenient :/ Since currently everything else but the database is logged when setting `--debug`, we should only see suggestions to add more `COMPONENT_DEBUG` settings if indeed `COMPONENT`'s debug log is so noisy that it needs to be split out and not part of default logging.

Reply:
Okay, renamed from `DEBUG_DATABASE` to `DEBUG_LOG_DATABASE`, so it doesn't look like its passed on to the db backend or something.